bootloader/src/pyi_launch.c

```diff
@@ -467,10 +469,12 @@ int pyi_launch_execute(ARCHIVE_STATUS *status)
     if (!*PI_Py_FileSystemDefaultEncoding) {
         char *saved_locale, *loc_codeset;
         saved_locale = strdup(setlocale(LC_CTYPE, NULL));
```

Comment:
Actually, I shouldn't say it's weird - [it's part of the C standard:](http://pubs.opengroup.org/onlinepubs/009695399/functions/setlocale.html)
```
If setlocale() is not invoked, the POSIX locale is the default at entry to main().
"C"
  Equivalent to "POSIX".
```